### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by likith012** [sthalles/PyTorch-BYOL#10]
I am not sure, but I think the loss function is a bit different from the original paper [https://github.com/sthalles/PyTorch-BYOL/blob/f007ed6b3619a3ae00eb9c95cde342eac5b98923/trainer.py#L35](here). In the paper the loss function equation is for unnormalized tensors(loss equation combines both normalization and l2 loss), but here you are normalizing them and applying the loss function.

![image](https://user-images.githubusercontent.com/14799306/142082581-724b9caa-b41a-4b42-8bfc-f3cbc9935f26.png)

**Comment by Issac-Sun** [comment]
I completed a simple implementation on CIFAR10 and the cosine similarity loss in the original article, you can follow https://github.com/Issac-Sun/BYOL_pytorch
